modules/user.py

```diff
@@ -142,8 +142,10 @@ def login(self, name=None, password=None, skey="", *args, **kwargs):
 		if not name or not password or not securitykey.validate(skey, useSessionKey=True):
 			return self.userModule.render.login(self.loginSkel())
 
+		name = name.lower().strip()
```

Comment:
Due to this assignment `fromClient` in L. 182 will not get the original value. 
Im not sure whether it breaks something, but maybe this could be an unwanted side effect.